### ISSUE TITLE
zabbix: add to installation section mkdir

### ISSUE THREAD

**Issue by bes-internal** [ugoviti/zabbix-templates#1]
`mkdir -p /etc/zabbix/zabbix_agentd.d/scripts/`

**Comment by bes-internal** [comment]
default /etc/zabbix/zabbix_agentd.conf does not have these lines
```
Include=/etc/zabbix/zabbix_agentd.d/
Include=/etc/zabbix/zabbix_agentd.d/*.conf
```
add instuctions for them

**Comment by ugoviti** [comment]
Thankyou for the issue... fixed in commit b740bea0ce76bb0502f0b5985f2e0b1aaf2a64a8

Kind Regards
